github/ci/prow-deploy/tasks/secrets.yml

```diff
@@ -96,3 +96,8 @@
   copy:
     content: '{{ windowsProductKeys["sysprep-001"] }}'
     dest: '{{ secrets_dir }}/win-sysprep-001'
+
```

Comment:
Besides putting in place the file with the secret we need to add an entry to customize secretGenerator here https://github.com/kubevirt/project-infra/blob/main/github/ci/prow-deploy/kustom/overlays/ibmcloud-production/kustomization.yaml#L269 so that the k8s secret is actually created.

Reply:
Added it to secretGenerators for both clusters. PTAL!